tests/manager/addCustomer/managerCanAddNewCustomer.spec.js

```diff
@@ -1,28 +1,22 @@
 import { test } from '@playwright/test';
 import { faker } from '@faker-js/faker';
+import { AddCustomerPage } from '../../../src/pages/manager/AddCustomerPage';
 
 test('Assert manager can add new customer', async ({ page }) => {
-/* 
-Test:
-1. Open add customer page by link https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager/addCust
-2. Fill the First Name.  
-3. Fill the Last Name.
-4. Fill the Postal Code.
-5. Click [Add Customer].
-6. Reload the page (This is a simplified step to close the popup)
-7. Click [Customers] button.
-8. Assert the First Name of the customer is present in the table in the last row. 
-9. Assert the Last Name of the customer is present in the table in the last row. 
-10. Assert the Postal Code of the customer is present in the table in the last row. 
-11. Assert there is no account number for the new customer in the table in the last row. 
+  const addCustomerPage = new AddCustomerPage(page)
```

Comment:
I would move constants outside of the test() function. Usually test should contain only test logic. Other things should be in before/after hooks or just in file